### PR TITLE
Answered requests list

### PR DESCRIPTION
**Pull request by CeliaBoiss**

### CODE REVIEW

**Review by KevinRoig — DISMISSED** [review]
Seems ok for me.

File: src/Controller/RequestController.php [hunk 1]
```diff
@@ -91,4 +90,59 @@ private function issetPost(): array
...
+    {
+        $errors = [];
+        $requests = [];
+        $users = [];
+        $answererId = [];
+
+        if ($_SERVER['REQUEST_METHOD'] === 'POST' && isset($_POST['userId'])) {
+            if ($_POST['userId'] !== 'Toutes les demandes') {
+                $answererId = trim($_POST['userId']);
+                $answererId = (int)$answererId;
```
Comment (RomainClair) [inline src/Controller/RequestController.php]: You should verify it's a int
You should use intval() instead of type casting

**Review by RomainClair — CHANGES_REQUESTED** [review]

**Review by KevinRoig — DISMISSED** [review]
OK

**Review by KevinRoig — APPROVED** [review]
Ok

**Review by RomainClair — APPROVED** [review]

**Pull request merged**